index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Poppins&family=Roboto+Flex:opsz,wght@8..144,500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+  <header>
+  <nav id="toolbar" class="nav-bar">
+      <span id="logo"><a href="https://github.com/Bludidi/Portfolio_and_mobile_first">BBL</a></span>
```

Comment:
- [ ] Good job wrapping your logo with the `a` tag. Your overall design is excellent but the only thing left is the styling of the logo. Your logo's current design and that of Figma's design do not tally. Please see the screenshots below:

👉  Please use the right styles declarations. Consider the following: 
- `text-decoration: none;`
- `font-family: use right font as per Figma`
- others

## Figma design
![](https://github.com/microverseinc/curriculum-html-css/raw/main/portfolio/images/m1_setup/toolbar-mobile.png)

## Your Design
![image](https://user-images.githubusercontent.com/84156714/185712539-8214436d-53b3-4d54-9f7a-df2b4499d4dd.png)

Reply:
I have updated the LOGO font family and the value of text-decoration is set to none.